tests/io/test.directory.js

```diff
@@ -198,3 +201,19 @@ describe('Directory.getFileAsString()', function() {
   });
 
 });
+
+
+describe('Directory.getChunkAsBuffer()', function() {
+
+  it('should get a buffer', () => {
+    var myDirectory = new Directory('tests/fixtures/io/');
+    return myDirectory.getFiles()
+      .then(() => {
+        return myDirectory.getChunkAsBuffer('dir2/dir3/file3.txt', 2);
+      })
+      .then((buffer) => {
+        assert.equal(buffer.toString(), '12');
```

Comment:
Ah, shoot, okay.

In my tests with fixture data (XPIs) I referenced what was inside the fixture data so the tests would be more immediately clear without having to reference the fixtures (eg https://github.com/mozilla/addons-linter/blob/master/tests/test.linter.js#L749). We should do that here just so it's immediately apparent.